.gitignore

```diff
@@ -7,8 +7,8 @@ node_modules
 *.vsix
 **/bin/
 **/obj/
-src/modules/samples/rust/target
-src/modules/samples/rust/Cargo.lock
+**/rust/target
```

Comment:
Will this target path work for all instances where Rust may be used? Ex: `src/modules/my-rust-module/target`

Reply:
This assumes that all rust components will be in a `rust` directory somewhere. Is that the expected pattern?